spec/services/github_user_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+describe GithubService do
+  it 'displays number of user repos' do
+    current_user = User.create(username: 'pixapi')
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(current_user)
+
+    repos = GithubRepo.repos(current_user)
+
+    expect(repos.count).to eq(6)
```

Comment:
Think there might be a piece missing here where you're either saving a token in your environment or otherwise stubbing out more information from your Service. Let's be sure to discuss during the eval.